config/alliance_fee_share.json

```diff
@@ -98,16 +118,17 @@
     }
   },
   "partner_fee_allocations": {
-    "default": {
-      "vebal_share_pct": 0.125,
-      "vote_incentive_pct": 0.35,
-      "partner_share_pct": 0.35,
-      "dao_share_pct": 0.175
-    },
-    "default_non_core": {
-      "vebal_share_pct": 0.475,
-      "partner_share_pct": 0.35,
-      "dao_share_pct": 0.175
+      "default": {
+        "vebal_share_pct": 0.0625,
+        "vote_incentive_pct": 0.70,
+        "partner_share_pct": 0.15,
+        "dao_share_pct": 0.0875
+      },
+      "default_non_core": {
+        "vebal_share_pct": 0.25,
+        "vote_incentive_pct": 0.0,
+        "partner_share_pct": 0.50,
+        "dao_share_pct": 0.25
```

Comment:
@Xeonus here is the split that these two pools will use. currently both are being identified as non-core